lib/note-detail.jsx

```diff
@@ -99,14 +104,36 @@ export default React.createClass( {
 						style={ divStyle }
 					>
 						<NoteContentEditor
-							ref={ this.saveEditorRef }
 							content={ content }
 							filter={ filter }
 							onChangeContent={ this.queueNoteSave }
+							ref={ this.saveEditorRef }
```

Comment:
is this change necessary? I know they are out of order alphabetically but I tend to put `ref` at the top as a pattern since it's a special prop. in any case, I think this change is unnecessary in this PR